pkg/dashboard/api.go

```diff
@@ -60,6 +65,22 @@ func contextSetter(data *objects.DataLayer) gin.HandlerFunc {
 	}
 }
 
+// Middleware for CORS
+func corsMiddleware() gin.HandlerFunc {
```

Comment:
This has to be gone in favor of `allowCORS()` function that is triggered via env variable. By that, there will be no security risk of forgetting dev code pieces.